Documentation/ceph-csi-troubleshooting.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Ceph-CSI Configuration
+weight: 1200
```

Comment:
This weight will add the entry in the TOC after the Ceph Tools --> Common Issues, and before the "Monitor Health" topic.
```suggestion
weight: 11125
```